Sources/PointFree/Database.swift

```diff
@@ -99,7 +99,7 @@ private func upsertUser(withGitHubEnvelope envelope: GitHub.UserEnvelope) -> Eit
       envelope.gitHubUser.name
     ]
     )
-    .flatMap { _ in fetchUser(gitHubAccessToken: envelope.accessToken) }
+    .flatMap { _ in AppEnvironment.current.database.fetchUserByGitHub(envelope.accessToken) }
```

Comment:
I pulled this in from my other PR.

not even sure what the future of our DB mock is, but either way seems like we should just always use the environment's database endpoints